metadata-ws/src/test/resources/features/accession.feature

```diff
@@ -0,0 +1,68 @@
+Feature: accession object
+
+  Scenario Outline: find different objects by accession and version
```

Comment:
I can see there are mixtures of test cases here: testing the validation of accessionVersionId and testing the searchability of different objects through accessionVersionId. Please separate them. Even though if separated, they may share a lot of steps to creating testing objects first, if you think that is a problem, deal with it in another way. Please don't mix them together.

Reply:
https://github.com/EBIvariation/metadata-ws/pull/46#discussion_r272830455